domain/bakerystorage/service/config.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package service
+
+import (
+	"context"
+
+	"github.com/go-macaroon-bakery/macaroon-bakery/v3/bakery"
+	"github.com/juju/errors"
+)
+
+type ConfigState interface {
+	InitialiseBakeryConfig(ctx context.Context, localUsersKey, localUsersThirdPartyKey, externalUsersThirdPartyKey, offersThirdPartyKey *bakery.KeyPair) error
+	GetLocalUsersKey(context.Context) (*bakery.KeyPair, error)
+	GetLocalUsersThirdPartyKey(context.Context) (*bakery.KeyPair, error)
+	GetExternalUsersThirdPartyKey(context.Context) (*bakery.KeyPair, error)
+	GetOffersThirdPartyKey(context.Context) (*bakery.KeyPair, error)
+}
+
+type ConfigService struct {
```

Comment:
Same with `Config` here.

Reply:
`BakeryConfigService` is what I have settled on, to be consistent with the state interface